tests/test_tracing.py

```diff
@@ -195,9 +209,9 @@ def test_with_complete_datadog_trace_headers(self):
             lambda_ctx,
         )
         self.assertEqual(source, "event")
-        self.assertDictEqual(
+        self.assertEqual(
             ctx,
-            {"trace-id": "123", "parent-id": "321", "sampling-priority": "1"},
+            Context(trace_id=123, span_id=321, sampling_priority=1)
```

Comment:
Probably best to do as you did previously, an assert each for trace id, span id, and sampling priority.